app/views/sections/_form.html.erb

```diff
@@ -8,6 +8,11 @@
     <%= f.label :week, 'Section week' %>
     <%= f.text_field :week, class: 'form-control' %>
   </div>
+  <div class="form-group">
+    <%= f.label :github_path, 'Section Github URL' %>
+    <%= f.text_field :github_path, class: 'form-control' %>
+  </div>
+  <p class="alert alert-info">Note that building a section from Github will take awhile.</p>
```

Comment:
https://www.quickanddirtytips.com/education/grammar/while-versus-awhile

Reply:
got it :)

Reply:
Also I'm thinking maybe I should move that into a background task. It'll need to notify Franz though if it fails.

Reply:
Yup, good idea. You can have it email him on failure or success. You could do something with websockets to notify through the web UI, but I don't know how difficult that would be. There must be a fairly standard way to do it.